### ISSUE TITLE
an arc slider

### ISSUE THREAD

**Issue by silencer07** [react-native-elements/react-native-elements#2542]
Would be nice if there is an option to arc the slider, values 0-89 degrees. The available arc sliders are too old already

**Describe alternatives you've considered**
react-native-circular sliders and the like

**Additional context**
https://www.google.com/search?q=react+native+arc+slider&rlz=1C5CHFA_enPH855PH855&sxsrf=ALeKk00wmtqkHDrTNwLSnCusecGFJt3InQ:1598949550885&tbm=isch&source=iu&ictx=1&fir=PJAuqBj0AUcKiM%252C-ppcnROG375rPM%252C_&vet=1&usg=AI4_-kQNx6ZnavuOBSu7N9Mm2CTGupY3sA&sa=X&ved=2ahUKEwiJ4Pmfx8frAhVLw4sBHY47DCcQ9QF6BAgMEB0#imgrc=vUYLa-B37mC5OM

**Comment by flyingcircle** [comment]
@silencer07 that sounds like an interesting suggestion. Maybe you can fork one of the outdated libraries, update it, and then we can pull it into RNE?

**Comment by pranshuchittora** [comment]
@flyingcircle I think we can pull off arc slider in RNE easily as that requires working over the SVG paths.
We can use the current slider to extend the functionality of our Arc Slider.

**Comment by vividh** [comment]
I'm interested to work on this. Any pointers?

> @flyingcircle I think we can pull off arc slider in RNE easily as that requires working over the SVG paths.
> We can use the current slider to extend the functionality of our Arc Slider.

Do you mean by adding a few props to the existing Slider component, it can be displayed as arc (something like `presentationMode={'arc' | 'line'}`), or should this be an entirely separate new component?

**Comment by flyingcircle** [comment]
If it is possible, I think it is usually better to have it be a separate component. If there's a way that you can find to reuse code between the 2 that'd be even better.

**Comment by ArenaGrenade** [comment]
Hi! @flyingcircle seems like nobody is working on this. I want to pick this up.

**Comment by ArenaGrenade** [comment]
> @flyingcircle I think we can pull off arc slider in RNE easily as that requires working over the SVG paths.
> We can use the current slider to extend the functionality of our Arc Slider.

I don't think this would be feasible as of the current state of the library as I don't see any SVG based code or am I missing something...
